fake/src/impls/mod.rs

```diff
@@ -1,6 +1,6 @@
 //! This module contains implementations of `Dummy` trait
 
-#[cfg(feature = "bigdecimal")]
+#[cfg(feature = "bigdecimal-rs")]
```

Comment:
This allows for the use of `bigdecimal-rs` directly without needing to enable `rust_decimal` as well.